### DISCUSSION TITLE
How to fully disable frontend routing

### DISCUSSION THREAD

**Discussion by CheetoTrump** [nuxt/nuxt#29958] [Questions]
Hello, im not really much of a frontend dev but i bought a comp kit that had a nuxt setup, and i made a frontend using nuxt i really liked but i am writing my backend using c++ and i want to completely disable routing or what ever is nessessary to stop the frontends behavior i.e

my backend serves /login/index.html on the / route and it loads perfectly fine *BUT* it adjusts the route to /login and i would like to deal with all the routing on backend for sessions etc

**Comment by ChrisGV04** [comment]
I'm not sure I understood your use case, but if you want your back-end to handle the HTML maybe you should not use Nuxt.

Nuxt uses Vue to render the front-end, and once the browser loads your app for the first time it will become a Single Page Application, which means that every time the user changes a page it will be done from the browser, not from the back-end.

If you need your C++ back-end to be the one that renders the html/css/js then you might be better off using something else. As far as I know, this is not possible with Nuxt. You might be able to do it with standalone Vue.js, but not with Nuxt.

**Comment by angelhdzmultimedia** [comment]
I'm not an expert, but I think what you need is `ssr: false` in the nuxt config to enable SPA.
